lib/attribute.ts

```diff
@@ -3,6 +3,7 @@
 
 import Long from 'long';
 import {onnx} from 'onnx-proto';
+import {onnxruntime} from './ort_format_schema';
```

Comment:
is it possible to change a exported name other than onnxruntime or to set an alias for onnxruntime.experimental.fbs?